Scripts/Mend SCA/generate-async-report.sh

```diff
@@ -101,13 +101,13 @@ if [[ "$MEND_REPORTSCOPE" = "Organization" ]] ; then
 	tokenScope=org
 	token=$MEND_ORG_UUID
 elif [[ "$MEND_REPORTSCOPE" = "Product" ]] ; then
-	[[ -z $WS_PRODUCTTOKEN ]] && echo "No WS_PRODUCTTOKEN specified and MEND_REPORTSCOPE=Product" && exit
+	[[ -z $MEND_PRODUCTTOKEN ]] && echo "No MEND_PRODUCTTOKEN specified and MEND_REPORTSCOPE=Product" && exit
 	tokenScope=product
-	token=$WS_PRODUCTTOKEN
+	token=$MEND_PRODUCTTOKEN
```

Comment:
This should be changed because of: https://github.com/mend-toolkit/mend-examples/blob/135433c0e8707ef21fb080853cad0d85578d3aab/Scripts/Mend%20SCA/generate-async-report.sh#L23
To use MEND_PRODUCT_UUID as it shows in line 23 of the file.

Reply:
Updated line 23 to match code.